classes/2.py

```diff
@@ -0,0 +1,89 @@
+class my_class(object):
```

Comment:
Стоило дать более осмысленное название, например
```suggestion
class RNA:
```

Reply:
Это относится и к следующим заданиям